.gitignore

```diff
@@ -12,7 +12,7 @@ msg/*Feedback.msg
 msg/*Goal.msg
 msg/*Result.msg
 msg/_*.py
-
+roslib.min.js
```

Comment:
you are ignoring all roslib.min.js files. Although only file in leobot_web_server need to be ignore.